App/components/Switch/__snapshots__/switch.test.tsx.snap

```diff
@@ -36,14 +28,33 @@ exports[`Switch component Component with only necessary props matches snapshot 1
   onResponderTerminationRequest={[Function]}
   onStartShouldSetResponder={[Function]}
   style={
-    Object {
-      "opacity": 1,
-    }
+    Array [
+      Object {
+        "backgroundColor": "#ededed",
```

Comment:
maybe we should use color from the theme?

Reply:
IDK, it's our default switch without any passed props